apps/ledger-live-desktop/src/renderer/hooks/storage-provider/useStorage.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { createContext, useContext } from "react";
+import { DesktopStorageProvider } from "./DesktopStorageProvider";
+
+const StorageContext = createContext(new DesktopStorageProvider());
+
+export const useStorageProvider = () => {
```

Comment:
[COULD] This hook seems like not to be used twice, so putting all the initialisation in the function at line 14 can make it more clear and simpler, WDYT ?

Reply:
I'm not sure I understand

Reply:
I've made a mistake.